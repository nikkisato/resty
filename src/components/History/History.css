.container{
    width: 400px;
    height: auto;
    min-height: 600px;
    margin-top:0;
    margin-left: 20px;
    border: solid 1px black;
}