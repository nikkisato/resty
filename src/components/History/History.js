import React from 'react';
import propTypes from 'prop-types';
import styles from './History.css';

export default function History({ history }) {
  const historyListItems = history.map(link => {
    <li key={link}>
      {link}
    </li>;
  });

  return (
    <div className={styles.history}>
      <h2>History Section</h2>
      <ul className={styles.historyList}>
        {historyListItems}
      </ul>


    </div>
  );
}

History.propType = {
  history: propTypes.array.isRequired
};
