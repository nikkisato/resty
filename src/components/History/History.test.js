import React from 'react';
import { shallow } from 'enzyme';
import History from './History';

describe('History component', () => {
  it('renders History', () => {
    const wrapper = shallow(<History history = {['Test']}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
