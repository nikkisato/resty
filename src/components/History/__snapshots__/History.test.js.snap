// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component renders History 1`] = `
<div
  className="container"
>
  <h3>
    History
  </h3>
</div>
`;
